app/templates/components/ui-table/cell/cell-roles.hbs

```diff
@@ -1 +1 @@
-<span>sample@gmail.com (Organizer)</span>
+<span>{{record.roles}}({{t 'Organizer'}})</span>
```

Comment:
@geekyd so for what I should check in assert.ok in the integration test for in cell-role-test.js